pyproject.toml

```diff
@@ -36,6 +36,7 @@ include = [
   "config/boot_config.txt",
   "config/support_tunnel_crontab",
   "config/support_group_sudoers",
+  "config/crontab",
```

Comment:
Wait how was this working before?

Reply:
`pyproject.toml` only really gets used for `./scripts/deploy` to ship things to the AmpliPi - in that case, it simply didn't. However, regular upgrades download a tarball from github that do not do this whitelisting, and thus any updates from a crontab would still get intsalled. (Don't look too closely at who last edited the cronttab before these changes :eyes: )